.golangci.toml

```diff
@@ -67,6 +67,7 @@ files = [
   "**/pkg/tsdb/parca/**/*",
   "**/pkg/tsdb/tempo/*",
   "**/pkg/tsdb/tempo/**/*",
+   "**/pkg/tsdb/cloudwatch/**/*",
```

Comment:
i don't know how exactly this thing works, but other datasources have 2 lines each... cloudwatch only 1. is this intentional?

Reply:
Fixed!